kore/src/Kore/Repl/Data.hs

```diff
@@ -390,6 +479,14 @@ helpText =
     \                                         <type> can be 'stderr' or 'file filename'\n\
     \                                         <switch-timestamp> can be enable-log-timestamps\
                                               \ or disable-log-timestamps\n\
+    \debug-[type]-equation [eqId1] [eqId2] .. show debugging information for multiple specific equations;\
+                                              \ [type] can be 'attempt' or 'apply', or nothing\
```

Comment:
I don't think anyone will get confused by this explanation, but when [type] is nothing then the remaining command is `debug--equation` (with two `-`)

Reply:
Yeah I wrote it like that initially, and thought it might actually be more confusing for a new user than the explanation above. If you think it's the other way around I can change it back.

Reply:
You're probably right